samba/docker-compose.yml

```
@@ -0,0 +1,30 @@
+version: '3.7'
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: samba_homepage_1
+      APP_PORT: 9445
+
+  homepage:
+    image: joseluisq/static-web-server:2@sha256:c8345c2c469ba249cbc986b6155dc275ce86bcd30e0b29755fda4deb501bbddd
```

Maybe try to reduce the permissions to a lower user to improve sandboxing of the container
```suggestion
    image: joseluisq/static-web-server:2@sha256:c8345c2c469ba249cbc986b6155dc275ce86bcd30e0b29755fda4deb501bbddd
    user: "1000:1000"
```

---

Also, please change the tag of `2` to `2.32.0`
```suggestion
    image: joseluisq/static-web-server:2.32.0@sha256:c8345c2c469ba249cbc986b6155dc275ce86bcd30e0b29755fda4deb501bbddd
```